conf.yaml

```diff
@@ -2089,6 +2092,27 @@ contents:
                 repo:   elasticsearch
                 path:   docs/Versions.asciidoc
                 exclude_branches:   [ 2.4, 2.3, 2.2, 2.1, 2.0, 1.7, 1.6, 1.5, 1.4, 1.3, 0.90 ]
+          # TODO: Remove this version of the Security guide once a redirect is
+          # in place from /en/endpoint to /en/security
+          - title:      Elastic Endpoint Security
```

Comment:
I just copied this from the current Elastic Endpoint Security book on the master branch.